src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  position: absolute;
```

Comment:
The 'position: absolute;' property for img might cause layout issues. Use 'align-self: center;' on the img element if it's inside a flex container.